tsconfig.json

```diff
@@ -1,12 +1,10 @@
 {
     "compilerOptions": {
         "sourceMap": true,
-        "allowJs": false,
-        "noImplicitAny": true,
+        "allowJs": true,
+        "noImplicitAny": false,
         "module": "commonjs",
-        "target": "es5",
-        "declaration": true,
-        "declarationDir": "types"
```

Comment:
It's not a good idea to remove declaration properties.
I think you broke the api autocomplete